### ISSUE TITLE
Document to be applicable to all versions of WCAG 2

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#260]
On 9 November TF decided we would make the WCAG2ICT note more explicitly cover all three WCAG versions. Mainly this was to enable us to provide unique guidance for WCAG 2.0 and 2.1 for 4.1.1 Parsing in this same document. This will require the following changes to the document:
- (NO CHANGE) All document links to WCAG criteria, understanding, etc. will remain linking to WCAG 2.2 as this is the latest information.
- [x] (SMALL CHANGE) Editors discussed: Do we mark sections for new criteria, guidelines, and terms added in WCAG 2.1 and 2.2 as such by saying when they were introduced? We decided it was better not to introduce that complexity to the document. Instead...Suggestion from Phil - just add a link to the [New features in WCAG 2.1](https://www.w3.org/TR/WCAG21/#new-features-in-wcag-2-1) and [New features in WCAG 2.2](https://www.w3.org/TR/WCAG22/#new-features-in-wcag-2-2) at some appropriate point in the document - such as the [Comparison with the 2013 WCAG2ICT Note](https://w3c.github.io/wcag2ict/#comparison-with-the-2013-wcag2ict-note).

Known updates needed:
- [x] Update the document title to "Guidance for Applying WCAG 2 to Non-web Documents and Software" - This may need to be done by Daniel, as I'm not sure what file this is in. (Done by Phil - edited title in index.html)
- [x] Abstract - Update the title, and add to description that this also covers WCAG 2.0 and 2.1 and add links to those recommendations similar to the "[WCAG22]" one that is already there. NOTE: You'll need to use markup like what is already used in WCAG 2.2 for this to work properly, which I think is to use [[WCAG21]] and [[WCAG 20]].
- [x] [Status of this document](https://github.com/w3c/wcag2ict/blob/mjm-all-wcag-2-versions/status-of-this-document.md) - Update the title
- [x] Content in the file [introduction.md](https://github.com/w3c/wcag2ict/blob/mjm-all-wcag-2-versions/introduction.md). Also update the section called Document Conventions to add a bullet saying "Hereafter, the use of WCAG 2 means all WCAG 2.x versions - 2.0, 2.1, and 2.2."
(Done by Phil, but I've left refs to WCAG 2.2 when they refer to a specific document e.g. Intent)
- [x] Throughout document, change any reference to "WCAG 2.2" to "WCAG 2" with the exception of directly named document links to that version.
- [x]  To section D.1 Informative References, add WCAG 2.0 and 2.1. 
    - [WCAG20]
[Web Content Accessibility Guidelines (WCAG) 2.0](https://www.w3.org/TR/WCAG20/). Ben Caldwell; Michael Cooper; Loretta Guarino Reid; Gregg Vanderheiden et al. W3C. 11 December 2008. W3C Recommendation. URL: https://www.w3.org/TR/WCAG20/
    - [WCAG21]
[Web Content Accessibility Guidelines (WCAG) 2.1](https://www.w3.org/TR/WCAG21/). Michael Cooper; Andrew Kirkpatrick; Joshue O'Connor; Alastair Campbell. W3C. 21 September 2023. W3C Recommendation. URL: https://www.w3.org/TR/WCAG21/

**Comment by maryjom** [comment]
@pday1 I had to implement the first checkbox item to get the last one.

**Comment by pday1** [comment]
@maryjom Changes committed to your PR. See what you think in the https://github.com/w3c/wcag2ict/blob/mjm-all-wcag-2-versions/status-of-this-document.md [status](https://github.com/w3c/wcag2ict/blob/mjm-all-wcag-2-versions/status-of-this-document.md) file, as I've left some refs to WCAG 2.2, as I thought that it was ambiguous to change to WCAG 2 when referring to specific documents.

**Comment by pday1** [comment]
@maryjom @ChrisLoiselle In answer to the question above: "Do we mark sections for new criteria, guidelines, and terms added in WCAG 2.1 and 2.2 as such by saying when they were introduced?"

I think this adds more complexity to the document (and more work for us). I think that it should be sufficient to just link to the WCAG 2.1 and WCAG 2.2 list of changes. 
[What's new in WCAG 2.1](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-21/)
[What's new in WCAG 2.2](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/)

**Comment by maryjom** [comment]
Since we've merged in the changes, this issue can be closed.
